db/migrate/20190715133218_create_resource_categories.rb

```diff
@@ -0,0 +1,16 @@
+class CreateResourceCategories < ActiveRecord::Migration[5.2]
+  def change
+    create_table :resource_categories do |t|
+      t.string  :name
+      t.text    :short_description
+      t.text    :description
+      t.binary  :icon
```

Comment:
I believe the idea was to store svg images directly in the database. I don't have a ton of experience doing that (my tendency is to store images in an S3 bucket or something and then just store a reference to them), but I read that it's doable if you store a binary string. Open to discussion and guidance here.